src/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+
+type BorderRadiusSize = "xxs" | "xs" | "s" | "m" | "l" | "xl" | "xxl";
```

Comment:
move all this to type.d.ts
i know u are waiting for @hdrayka9060 pr to be merged.
still move this to type.d.ts
it will be easy to migrate standards

Reply:
done